portal/src/main/webapp/js/src/study-view/component/MutatedGeneCNATable.js

```diff
@@ -218,6 +218,14 @@ var EnhancedFixedDataTableSpecial = (function() {
                     var list = ($("#hide_column_checklist").dropdownCheckbox("items"));
                     hideColumns(list);
                 });
+            // add title attributes to unlabeled inputs generated by
+            // .dropdownCheckbox() for accessibility, until
+            // https://github.com/Nelrohd/bootstrap-dropdown-checkbox/issues/33 is
+            // fixed upstream
+            $('#hide_column_checklist input[type="checkbox"].checkbox-all')
+                .attr('title', 'Select all');
+            $('#hide_column_checklist input[type="text"].search')
+                .attr('title', 'Search');
```

Comment:
@zhx828: may I ask how you updated this file? I do not see this file in the cBioPortal/enhanced-fixed-datatable repository. In pull request #1494 I proposed some minor changes to parts of this file that are specific to cBioPortal, did I apply those in the right place?